.docker/Dockerfile-build

```diff
@@ -1,4 +1,5 @@
-FROM golang:1.16-alpine AS builder
+# syntax = docker/dockerfile:1-experimental
```

Comment:
Is this needed?

Reply:
As far as I know this changes the way dockerfiles are parsed and it's still needed in the current version